include/object_fields.h

```diff
@@ -483,19 +483,19 @@
 #define /*0x0F4*/ oFireSpitterScaleVel OBJECT_FIELD_F32(0x1B)
 
 /* Blue Fish */
-#define /*0x0F4*/ oBlueFishUnkF4  OBJECT_FIELD_F32(0x1B)
-#define /*0x0F8*/ oBlueFishUnkF8  OBJECT_FIELD_S32(0x1C)
-#define /*0x100*/ oBlueFishUnk100 OBJECT_FIELD_F32(0x1E)
+#define /*0x0F4*/ oBlueFishRandomVel   OBJECT_FIELD_F32(0x1B)
+#define /*0x0F8*/ oBlueFishRandomTime  OBJECT_FIELD_S32(0x1C)
+#define /*0x100*/ oBlueFishRandomAngle OBJECT_FIELD_F32(0x1E)
 
 /* Fish Group */
-#define /*0x0F4*/ oFishGroupUnkF4  OBJECT_FIELD_F32(0x1B)
-#define /*0x0F8*/ oFishGroupUnkF8  OBJECT_FIELD_F32(0x1C)
-#define /*0x0FC*/ oFishGroupUnkFC  OBJECT_FIELD_F32(0x1D)
-#define /*0x100*/ oFishGroupUnk100 OBJECT_FIELD_S32(0x1E)
-#define /*0x104*/ oFishGroupUnk104 OBJECT_FIELD_F32(0x1F)
-#define /*0x108*/ oFishGroupUnk108 OBJECT_FIELD_F32(0x20)
-#define /*0x10C*/ oFishGroupUnk10C OBJECT_FIELD_F32(0x21)
-#define /*0x110*/ oFishGroupUnk110 OBJECT_FIELD_F32(0x22)
+#define /*0x0F4*/ oFishWaterLevel      OBJECT_FIELD_F32(0x1B)
+#define /*0x0F8*/ oFishPosY            OBJECT_FIELD_F32(0x1C)
+#define /*0x0FC*/ oFishRandomOffset    OBJECT_FIELD_F32(0x1D)
+#define /*0x100*/ oFishRandomSpeed     OBJECT_FIELD_S32(0x1E)
+#define /*0x104*/ oFishRandomDistance  OBJECT_FIELD_F32(0x1F)
+#define /*0x108*/ oFishRandomVel       OBJECT_FIELD_F32(0x20)
+#define /*0x10C*/ oFishRandomYDistance OBJECT_FIELD_F32(0x21)
+#define /*0x110*/ oFishRandomDistance2 OBJECT_FIELD_F32(0x22)
```

Comment:
Again, I don't like RandomDistance and RandomDistance2. Also you have RandomYDistance which is kinda confusing when reading code at a glance, since there are 3 fields which are named almost the same thing